<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OurChat</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(246, 246, 246);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="signin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 180, 0);
color: rgb(245, 245, 245);</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="shortcut">
     <string>Return, Enter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_text">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>帐号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_text">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>235</y>
      <width>140</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_signup">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(78, 96, 255);
border-style:none;
background:transparent;
</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="icon_button">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>70</y>
      <width>90</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>chat.jpg</normaloff>chat.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>120</width>
      <height>120</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_pushButton">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>0</y>
      <width>35</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logo_Button">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:none;
background:transparent;
color: rgb(140, 140, 140);</string>
    </property>
    <property name="text">
     <string>OurChat</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
